src/api.js

```diff
@@ -378,8 +378,11 @@
   Agent.prototype.getEndpoints = function (queueARNs, callbacks, pageInfoIn) {
     var self = this;
     var client = connect.core.getClient();
-    var pageInfo = pageInfoIn || { endpoints: [] };
+    connect.assertNotNull(callbacks, "callbacks");
+    connect.assertNotNull(callbacks.success, "callbacks.success");
+    var pageInfo = pageInfoIn || { };
 
+    pageInfo.endpoints = pageInfo.endpoints || [];
```

Comment:
Otherwise, this throws an error: `agent.getEndpoints(queueARNs, { success: (result) => { /* ... */ } });`